netbox/dcim/tables/devices.py

```diff
@@ -143,9 +143,8 @@ class Meta(NetBoxTable.Meta):
 class DeviceTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable):
     name = tables.TemplateColumn(
         verbose_name=_('Name'),
-        accessor=Accessor('label'),
         template_code=DEVICE_LINK,
-        linkify=True
+        linkify=True,
```

Comment:
Oops, forgot I added this when adding the previous `order_by` argument. I'm a monster!